src/orchard/indent.clj

```diff
@@ -0,0 +1,173 @@
+(ns orchard.indent
+  "`:style/indent` inference."
+  (:require
+   [clojure.set :as set]
+   [clojure.string :as string])
+  (:import
+   (java.util List)))
+
+(defn- index-by-name [m]
+  (into {}
+        (map (fn [[k v]]
+               [(-> k name) [k v]]))
+        m))
+
+(def clojure-mode-indents-exact
+  "`:style/indent` rules based on clojure-mode, for names that are short
+  and might be a syllable/word within a larger, unrelated word/phrase.
+
+  e.g. `are` can be part of `compare`, so `are` should only be looked for exact matches,
+  not for fuzzy matches."
+  (index-by-name '{clojure.test/are          2
+                   clojure.core/binding      1
+                   clojure.core/case         1
+                   catch                     2
+                   clojure.core/comment      0
+                   clojure.core/cond         0
+                   def                       :defn
+                   clojure.core/defn         :defn
+                   clojure.core/delay        0
+                   do                        0
+                   clojure.core/extend       1
+                   clojure.spec.alpha/fdef   1
+                   clojure.core/fn           :defn
+                   clojure.core/for          1
+                   clojure.core.async/go     0
+                   if                        1
+                   clojure.core/let          1
+                   clojure.core/locking      1
+                   clojure.core/loop         1
+                   clojure.core/ns           1
+                   clojure.core/proxy        [2 nil nil [:defn]]
+                   clojure.core/reify        [:defn [1]]
+                   clojure.test/testing      1
+                   this-as                   1
+                   clojure.core.async/thread 0
+                   try                       0
+                   clojure.core/when         1
+                   clojure.core/while        1}))
+
+(def clojure-mode-indents-fuzzy
```

Comment:
I don't get the need for fuzzy matches, as I think `clojure-mode` only does exact matches. Can you elaborate on this?

Reply:
That's fair, but let's just document this better.